src/EditorFeatures/Core/Shared/Utilities/ForegroundThreadAffinitizedObject.cs

```diff
@@ -66,19 +66,14 @@ internal static ForegroundThreadData CurrentForegroundThreadData
             }
         }
 
-        internal ForegroundThreadData ForegroundThreadData
-        {
-            get { return _foregroundThreadData; }
-        }
-
         internal Thread ForegroundThread
```

Comment:
I'd avoid it given where we are in the cycle since we might have external consumers.